tests/system/fim/common_tasks/files_configuration.json.j2

```diff
@@ -4,9 +4,9 @@
     "folder_length": 5,
     "file_length": 5,
     "file_size_specifications":[
-        { "size": 10240, "amount": 4000},
```

Comment:
Please restore it to default value

Reply:
Sure! thanks, https://github.com/wazuh/wazuh-qa/pull/570/commits/5096f007c4aeffc977ff346e4b8623baee3989d0